phpstan-baseline.neon

```diff
@@ -8470,6 +8470,11 @@ parameters:
 			count: 1
 			path: src/lib/REST/Input/Parser/Operation.php
 
+		-
```

Comment:
Added to baseline because Resolver is already checked by `ApplicationConfigRestResolverRegistry::hasResolver` method: https://github.com/ibexa/admin-ui/pull/1027/files#diff-37e803e13a8b6b5d2ef01e755fb2b0591ebde3fe95bbca5869b345c669c730e9R62-R66